Taskfile.yml

```diff
@@ -15,11 +15,9 @@ tasks:
   ci:
     desc: Workflow to run in CI
     cmds:
-      - task: workspace
+      - task: setup
```

Comment:
I missed this earlier. It might be worth adding a comment but we want `workspace` instead of `setup` here.
`setup` is for local development and includes installing `changie`. That's why there's the duplicate 
```yaml
- task: install-gofumpt
- task: install-golangci-lint
```
in `task ci` and also in `task setup`